build.yml

```diff
@@ -18,4 +18,3 @@ steps:
     repository: 'vscode-linux-build-agent'
     tags: ${{ parameters.arch }}
     command: push
-  condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```

Comment:
We should not merge this! We only want to push to the registry if the build happens from `master`.

Reply:
Yup was gonna revert before merging, need builds from this branch for testing :)